mappings/net/minecraft/util/math/Box.mapping

```diff
@@ -78,6 +78,12 @@ CLASS net/minecraft/class_238 net/minecraft/util/math/Box
 	METHOD method_29968 from (Lnet/minecraft/class_243;)Lnet/minecraft/class_238;
 		ARG 0 vec3d
 	METHOD method_30048 of (Lnet/minecraft/class_243;DDD)Lnet/minecraft/class_238;
+		ARG 0 center
+		ARG 1 dx
+		ARG 3 dy
+		ARG 5 dz
+	METHOD method_33660 smallestBoxContaining (Ljava/lang/Iterable;)Lnet/minecraft/class_238;
```

Comment:
Can we call it like `encompass`?

Reply:
```suggestion
	METHOD method_33660 encompass (Ljava/lang/Iterable;)Lnet/minecraft/class_238;
```

Reply:
Or possibly `enclose`. You should look up the dictionary and find the suitable one. If you think the vocabulary is obscure feel free to add some javadocs.

Reply:
We went over a few names in the discord before, but couldn't really agree on one. I was also considering `encapsulate`

Reply:
Hmm, I think we should go with `enclosing`

Reply:
It is static. Do you mean we should use `enclose` or actually `enclosing`.

Reply:
I mean `Box.enclosing(boxList)` looks natural code-wise.